src/test/extension.test.ts

```diff
@@ -6,12 +6,16 @@ const sourcegraph = createMockSourcegraphAPI()
 // For modules importing Range/Location/URI/etc
 mock('sourcegraph', sourcegraph)
 
-import { decorateEditor, decorateLine, getDecorations } from '../extension'
+import { activate, decorateEditor, decorateLine, getDecorations } from '../extension'
 
 describe('extension', () => {
     it('works', () => void 0)
 })
 
+describe('check for extension activation', () => {
+    it('activate extension', () => expect(activate(sourcegraph.ExtensionContext)).toEqual(void 0))
```

Comment:
`activate` is not called with `sourcegraph.ExtensionContext` - that class doesn't actually exist, it's just an interface, and an instance of that interface is passed